PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,22 @@
 <!-- Your title above should be a short description of what
 was changed. Do not include the issue number in the title. -->
 
+### References to other Issues or PRs
+<!-- If this pull request fixes an issue, write "Fixes #NNNN" in that exact
+format, e.g. "Fixes #1234". See
+https://github.com/blog/1506-closing-issues-via-pull-requests .-->
 
-<!-- Briefly explain what this pull request changes in each line below, with appropriate heading—'M' for major changes, 'm' for minor changes, 'n' for new features, and 'b' for backwards compatibility breaks and deprecations—and a colon(:), e.g., "M: Added changelogs". Write '[skip changelog]' if changes are trivial or not related to SymPy itself. -->
-## This PR changes
 
-<!-- [CHANGELOG END] -->
-<!-- If this pull request fixes an issue please indicate which issue by typing "Fixes #NNNN" below. -->
+### Brief description of what is fixed or changed
+<!-- This will be added to release notes automatically. If changes are trivial
+or not related to SymPy itself, please uncomment following line.-->
```

Comment:
Should "uncomment following line" be explained in more detail?

Reply:
I agree. People might not know how HTML comments work. They might even think the  [] is the comment.